submit.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+import paramiko
+
+import flask
+from flask import request
+
+from shlex import quote
+
+blueprint = flask.Blueprint('submit', __name__, url_prefix="/submit")
+
+ip = "10.168.0.42"
+keyPath = "./obabel-key"
+
+@blueprint.route("/", methods=['PATCH'])
+def submit():
```

Comment:
Can you look into making use of Mocks and add a test for this?  You could potentially create a mock object of the paramiko ssh client, then mock the return value of 'exec_command', (passing in whatever fake obabeCommand) and asserting the expected response from making an http post request with the submit route.

https://realpython.com/python-mock-library/#what-is-mocking
https://docs.python.org/3/library/unittest.mock.html

There are example usage here:
https://realpython.com/testing-third-party-apis-with-mocks/